strings/loot_data/tags/gun_upgrade.txt

```diff
@@ -13,6 +13,7 @@
 /obj/item/weapon/gun_upgrade/mechanism/overshooter    blacklisted=0    spawn_value=4.54545   spawn_price=100   prob_accompanying_obj=100    accompanying_objs=nothing
 /obj/item/weapon/gun_upgrade/mechanism/glass_widow    blacklisted=1    spawn_value=1.92308   spawn_price=100   prob_accompanying_obj=100    accompanying_objs=nothing
 /obj/item/weapon/gun_upgrade/mechanism/weintraub    blacklisted=0    spawn_value=3.125   spawn_price=100   prob_accompanying_obj=100    accompanying_objs=nothing
+/obj/item/weapon/gun_upgrade/mechanism/funswitch    blacklisted=0    spawn_value=3.125   spawn_price=100   prob_accompanying_obj=100    accompanying_objs=nothing 
```

Comment:
This is an automatically generated file and doesn't need to be changed for this PR.

Reply:
Should hopefully be removed from the PR.